### DISCUSSION TITLE
Change to some API parameters? / Deprecation warnings

### DISCUSSION THREAD

**Discussion by jat255** [usnistgov/NEMO#160] [General]
Hi @rptmat57,

I noticed an unexpected failure recently in our test code for NexusLIMS, related to the `username=` parameter no long working for the `/users` API endpoint. After some investigation, I noticed the parameter for getting a user by username is now `username__iexact=` 

This is a minor change on our side to support, so it's an easy fix, but I was more wondering if I missed an announcement somewhere that the API had changed. Our server was a bit behind in upgrades, so I think the change actually happened in 4.5.0 with the API expansion, but I didn't see any notes about parameters changing or deprecated API functionality.

Your release notes are already so awesome I hesitate to make any suggested changes, but if I could make one, it would be to specifically call out "deprecations" or breaking API changes so third-party connection maintainers can get a heads-up on these sorts of changes.

Thanks again for an awesome software!

**Comment by rptmat57** [comment]
I agree 100%. I will add a new API section to the release notes for any changes in API stuff.
Also, I will put the exact filter back since I believe wherever iexact is used we should also have exact

**Reply by jat255** [reply]
Great, thanks!
